include/modem/lte_lc.h

```diff
@@ -639,10 +663,42 @@ enum lte_lc_neighbor_search_type {
 	 *  and the search is performed for all supported bands.
 	 */
 	LTE_LC_NEIGHBOR_SEARCH_TYPE_EXTENDED_COMPLETE = 2,
+
+	/** GCI search, option 1. Modem searches EARFCNs based on previous cell history. */
+	LTE_LC_NEIGHBOR_SEARCH_TYPE_GCI_1 = 3,
```

Comment:
The GCI search types could be named similarly as the non-GCI search types, i.e. `LTE_LC_NEIGHBOR_SEARCH_TYPE_GCI_DEFAULT`, `LTE_LC_NEIGHBOR_SEARCH_TYPE_GCI_EXTENDED_LIGHT` and `LTE_LC_NEIGHBOR_SEARCH_TYPE_GCI_EXTENDED_COMPLETE`. The differences between different GCI search types looks very similar to non-GCI search types.

Reply:
I like that naming. Changed.